packages/components/organisms/f-header/src/assets/scss/navigation.scss

```diff
@@ -166,6 +166,10 @@ $nav-link-focus-color              : $color-focus;
             text-decoration: underline;
         }
     }
+
+    @include media('<=mid') {
+        color: $nav-text-color;
```

Comment:
What's the reason for only applying this colour on mobile? Does the colour change between mobile/desktop?